webrtc.nimble

```diff
@@ -36,5 +44,10 @@ proc runTest(filename: string) =
   exec excstr & " -r " & " tests/" & filename
   rmFile "tests/" & filename.toExe
 
-task test, "Run test":
+task test, "Run the test suite":
   runTest("runalltests")
+  exec "nimble build_example"
```

Comment:
not sure this should be part of the test

Reply:
I agree, I created a "Build examples" task.